package.json

```diff
@@ -49,8 +49,7 @@
     "vue": "3.2.39"
   },
   "devDependencies": {
-    "@babel/core": "7.19.1",
-    "@babel/eslint-parser": "7.19.1",
+    "@babel/eslint-parser": "^7.19.1",
```

Comment:
All versions in package.json must be fixed because we don't trust npm community. We've seen so many breaking changes shipped as minors that we have everything fixed. I think we should enforce that in the `.npmrc` and add `save-exact=true`.